linode/manager
# refactor: [M3-6350] - MUI v5 Migration - `Components > InlineMenuAction`

## Description 📝
Refactors `InlineMenuAction` by removing jss and cleaning up the overall usage of the component.

## Major Changes 🔄
- `Drawer.jsx` now using `IconButton` instead of `Button` for the close component
- Removed excessive styles for `InlineMenuAction` component
   - Created a `StyledActionButton` until we can figure out what we want to do with this type of button. It doesn't follow our Button standards.
- Updated imports/exports
- `Button.tsx`: `compactX` and `compactY` should be something that's capable of being applied to any button type. In addition, we need UX to look at adding padding for when we use these props. At times the button text comes awfully close to the edge of the button which is not ideal.
   - I removed these props to anything that wasn't applied to `buttonType='secondary'` since it had no effect previously, it was essentially a dead prop.

## Preview 📷

| Before  | After   |
| ------- | ------- |
| ![Screenshot 2023-06-15 at 4 33 57 PM](https://github.com/linode/manager/assets/125309814/7719dc67-6bf9-48e2-981c-ba8850a77586) | No Change |
| ![Screenshot 2023-06-15 at 4 33 47 PM](https://github.com/linode/manager/assets/125309814/41583d28-d435-4ea8-a1e1-f18c40f0d70e) | No Change |
| ![4](https://github.com/linode/manager/assets/125309814/b54a5221-1254-45e4-b67b-5b5d1984ba2a) | ![6](https://github.com/linode/manager/assets/125309814/9c4cbd47-7508-41a6-b868-5ec6b5ffd941) |

## How to test 🧪
1. Ensure there's no discrepencies for InlineMenuAction throughout site
2. Go to: http://localhost:3000/domains, http://localhost:3000/account/billing
